themes/origins/assets/component-slider.js

```diff
@@ -180,7 +191,10 @@ if (!customElements.get("yc-slider")) {
     reset(new_index = this.index) {
       this.index = new_index;
       this.move();
-      this.setArrows();
+
+      if (this.isArrowsVisible) {
+        this.setArrows();
+      }
```

Comment:
just saving a bit of space
```suggestion
      if (this.isArrowsVisible) this.setArrows();
```

Reply:
:nopers:

Reply:
maybe `this.isArrowsVisible && this.setArrows();`